docs/content/index.yml

```diff
@@ -32,6 +32,13 @@ tail: |
   on [Libera.Chat](https://libera.chat/). For more interactive discussions, feel free to join our [Discord server](https://discord.gg/yg6yjNmgAC).
 
 news:
+  - date: 7 September 2023
+    body: |
+      After a five-year hiatus, we've returned with a revitalized [GitHub organization](https://github.com/jqlang)
```

Comment:
I copied mostly from https://github.com/jqlang/jq/releases/tag/jq-1.7. Let me know if there is anything else you want to add